nemo/collections/nlp/data/dialogue/dataset/dialogue_bert_dataset.py

```diff
@@ -116,8 +158,103 @@ def __getitem__(self, idx):
             np.array(self.all_subtokens_mask[idx]),
             self.all_intents[idx],
             np.array(self.all_slots[idx]),
+            np.array(self.all_bio_slots[idx]),
+            np.array(self.bio_mention_labels[idx]),
+            np.array(self.mention_loss_mask[idx]),
         )
 
+    @staticmethod
+    def get_bio_slot_label_from_sequence(slot_label_list, label_of_other_type):
```

Comment:
the var label_of_other_type is not a very good name as it is not informative at all (maybe use something like label_id_for_empty_slot)